test/SearchResults-test.js

```diff
@@ -7,7 +7,7 @@ const sinon = require('sinon');
 const SearchResults = require('../components/SearchResults');
 
 describe('<SearchResults />', function() {
-  it('should render an <ul />', function() {
+  it('should render an <ul>', function() {
```

Comment:
LGTM 👍  If this was a component though I would add those self closing tags. Example `<AutoComplete />`